### PR TITLE
initial draft of general python guide

### PR DESCRIPTION
**Pull request by ChristinaLK**
Waiting for comments and final read-through.

### CODE REVIEW

File: start/software/manage-python-packages.md [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+
+Load Python.  
+
+     $ module load python/3.7.0
+
+Now, we will go through the steps to create a virtual environment.  The first 
+command creates the base environment. You 
+can swap out `my_env` for a more descriptive name like `scipy` or `word-analysis`.
+
+     $ python3 -m venv testenv
```
Comment (CarrieBrown) [inline start/software/manage-python-packages.md]: This should probably say `python3 -m venv my_env` to match with the rest of the document (or switch all references of `my_env` to say `testenv`)

File: start/software/manage-python-packages.md [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+	tar -xzf my_env.tar.gz
+
+	# Create the virtual environment on the remote hosts (redefines the env variables)
+	virtualenv-2.7 nltk_env
+
+	# Activate virtual environment
+	source my_env/bin/activate
+
+	# Run the Python script 
+	python3 myscript.py
```
Comment (CarrieBrown) [inline start/software/manage-python-packages.md]: Change `myscript.py` to say `script.py` to match the submit file.

File: start/software/manage-python-packages.md (hunk 1 above)
Comment (ChristinaLK) [reply start/software/manage-python-packages.md]: oops, you're right.

File: start/software/manage-python-packages.md (hunk 2 above)
Comment (ChristinaLK) [reply start/software/manage-python-packages.md]: thanks!

**Comment by ChristinaLK** [comment]
Going to resubmit this as a proper PR from my fork.

**Pull request closed without merging**